civihr_employee_portal/templates/civihr-employee-portal-custom-reports.tpl.php

```diff
@@ -105,6 +105,9 @@ class="btn btn-lg btn-secondary-outline text-uppercase">
             </div>
         </footer>
     </section>
+    <div data-graph-section="summary">
```

Comment:
Hmm... is it okay to have a table and headers in HTML and then just .append() tbody and thead to it?